pkg/cmd/cli/cmd/newapp.go

```diff
@@ -132,11 +132,61 @@ type NewAppOptions struct {
 	LogsForObject LogsForObjectFunc
 }
 
+type NewAppOptions struct {
+	AppOptions *BaseOptions
+}
+
+//Complete set all common default options for commands (new-app and new-build)
+func (o *BaseOptions) Complete(baseName, commandName string, f *clientcmd.Factory, c *cobra.Command, args []string, in io.Reader, out, errout io.Writer) error {
```

Comment:
@csrwng maybe the "base" Complete method and struct should be moved to its own file / existing helper file?